experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -203,6 +211,44 @@ export interface MapNode<TSchema extends MapSchema> extends TreeNode {
 	 */
 	get(key: FieldKey): TypedField<TSchema["mapFields"]>;
 
+	/**
+	 * Returns an iterable of keys in the map.
+	 *
+	 * @remarks
+	 * All fields under a map implicitly exist, but `keys` will yield only the keys of fields which contain one or more nodes.
+	 */
+	keys(): IterableIterator<FieldKey>;
+
+	/**
+	 * Returns an iterable of values in the map.
+	 *
+	 * @remarks
+	 * All fields under a map implicitly exist, but `values` will yield only the fields containing one or more nodes.
+	 */
+	values(): IterableIterator<UnboxField<TSchema["mapFields"]>>;
+
+	/**
+	 * Returns an iterable of key, value pairs for every entry in the map.
+	 *
+	 * @remarks
+	 * All fields under a map implicitly exist, but `entries` will yield only the entries whose fields contain one or more nodes.
+	 */
+	entries(): IterableIterator<[FieldKey, UnboxField<TSchema["mapFields"]>]>;
+
+	/**
+	 * Executes a provided function once per each key/value pair in the map, in insertion order.
+	 * @param callbackFn - the function to run for each map entry
```

Comment:
Nit: I generally recommend standard sentence casing for param comments and the like, since the generated docs are formatted assuming complete sentences (i.e. capitalizing the first letter of the first word in the comment)

Reply:
Makes sense to me, I'll do that going forward.